pyartcd/pyartcd/pipelines/build_sync.py

```diff
@@ -0,0 +1,316 @@
+import asyncio
+import glob
+import json
+import os
+
+import click
+import yaml
+
+from pyartcd.cli import cli, pass_runtime, click_coroutine
+from pyartcd.runtime import Runtime
+from pyartcd import exectools
+from pyartcd.util import branch_arches
+
+GO_ARCH_SUFFIXES = {
```

Comment:
Let's use `go_suffix_for_arch` from `doozerlib.util`

Reply:
Fixed in 76c8732146784ad4b64ee0cc39bae1ac2477ce43